src/cluster/src/server.rs

```diff
@@ -175,6 +175,12 @@ where
                 idle_merge_effort: Some(isize::cast_from(config.idle_arrangement_merge_effort)),
             },
         );
+        if let Some(cluster_size) = &config.cluster_size {
+            worker_config.set(
+                "materialize/storage.cluster_size".to_string(),
+                cluster_size.to_owned(),
+            );
+        }
```

Comment:
This function should only be used to initialize Timely and Differential, but not Materialize-specific parameters. Persist seems to be an acceptable exception here because it's used from everywhere.

Reply:
Ack